hieradata_aws/class/integration/feedback.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 
-govuk::apps::feedback::redis_host: 'backend-redis'
```

Comment:
Are these variables / file necessary, as they're already defined in [hieradata_aws/integration.yaml](https://github.com/alphagov/govuk-puppet/pull/11672/files#diff-769a9254659e825834006a559d95c14cdc1292e0a48758685f42b4d99dafd296)?

Reply:
Good question! Probably not. This was where I originally put the variables, but I think maybe neither of those places is right - it should probably be in hieradata_aws/class/integration/frontend.yaml. Let me check with someone who might know what the ideal is here.

Reply:
Fixed - by moving it into the right file.